syntax = "proto3";

package model;

option java_package = "com.stockearte.model";
option java_outer_classname = "StoreProto";

import "google/protobuf/empty.proto";
import "google/protobuf/wrappers.proto";

service StoreService {
  rpc GetStore(Store) returns (Store) {}
  rpc FindAll(google.protobuf.Empty) returns (Stores) {}
  rpc SaveStore(Store) returns (Store) {}
}

message Stores {
  repeated Store store = 1;
}

message Store {
  int32 idStore = 1;
  string storeName = 2;
  string code = 3;
  string address = 4;
  string city = 5;
  string state = 6;
  bool enabled = 7;
}